### PR TITLE
release(v1.10.0-alpha.0): prepare release

### PR DESCRIPTION
**Pull request by smira**
This is the official v1.10.0-alpha.0 release.

### CODE REVIEW

**Review by DmitriyMV — APPROVED** [review]

File: internal/app/machined/pkg/system/runner/process/process_test.go [hunk 1]
```diff
@@ -223,6 +223,10 @@ func (suite *ProcessSuite) TestStopSigKill() {
 }
 
 func (suite *ProcessSuite) TestPriority() {
+	if os.Geteuid() != 0 {
+		suite.T().Skip("skipping test, need root privileges")
+	}
```
Comment (dsseng) [inline internal/app/machined/pkg/system/runner/process/process_test.go:228]: Does it break w/o this?

**Review by dsseng — APPROVED** [review]

File: internal/app/machined/pkg/system/runner/process/process_test.go (hunk 1 above)
Comment (smira) [reply internal/app/machined/pkg/system/runner/process/process_test.go:228]: it breaks outside of container, requires more privs. I do smoke test via `go test ./...`

Comment (dsseng) [reply internal/app/machined/pkg/system/runner/process/process_test.go:228]: hm, okay, maybe on different systems it's different

**Review by frezbo — APPROVED** [review]

**Pull request merged**